src/Forms/Components/DropInAction.php

```diff
@@ -39,7 +39,11 @@ public function getExecutableActions(bool $reevaluate = false): array
         $this->evaluatedActions = [];
 
         foreach ($this->actions as $action) {
-            $this->evaluatedActions[] = $this->evaluate($action)?->component($this);
+            $actions = $this->evaluate($action);
+
+            foreach (Arr::wrap($actions) as $action) {
```

Comment:
Please change to the `$action` variable to `$nestedAction` for this loop so it doesn't conflict with the outer loop.

Reply:
Good call, done.